datacapturing/src/movebis/java/de/cyface/datacapturing/MovebisDataCapturingService.java

```diff
@@ -225,6 +225,9 @@ public void registerJWTAuthToken(final @NonNull String username, final @NonNull
             throws SynchronisationException {
         final AccountManager accountManager = AccountManager.get(getContext());
 
```

Comment:
I added this in case there are problems which we cannot reproduce in our app. We probably do not need to change anything so I didn't want to start refactoring when there is no need.